docs/tutorials/07_pegasos_qsvc.ipynb

```diff
@@ -7,9 +7,13 @@
    "source": [
     "# Pegasos Quantum Support Vector Classifier\n",
     "\n",
-    "There's another SVM based algorithm that benefits from the quantum kernel method. Here, we introduce an implementation of a another classification algorithm, which is an alternative version to the `QSVC` available in Qiskit Machine Learning and shown in the [\"Quantum Kernel Machine Learning\"](./03_quantum_kernel.ipynb) tutorial. This classification algorithm implements the Pegasos algorithm from the paper \"Pegasos: Primal Estimated sub-GrAdient SOlver for SVM\" by Shalev-Shwartz et al., see: https://home.ttic.edu/~nati/Publications/PegasosMPB.pdf.\n",
+    "## Overview\n",
+    "In this tutorial, we will implement another SVM based algorithm that benefits from the quantum kernel method, namely `PegasosQSVC` from Qiskit Machine Learning.\n",
```

Comment:
I would suggest to always use either Qiskit Machine Learning or `qiskit-machine-learning` (leaning against the second one)